cgmanifest.json

```diff
@@ -4955,8 +4955,8 @@
         "type": "other",
         "other": {
           "name": "python-py",
-          "version": "1.6.0",
-          "downloadUrl": "https://github.com/pytest-dev/py/archive/1.6.0.tar.gz"
```

Comment:
Shouldn't we update this line to use PyPI as well?

Reply:
You're probably right, since that's what we're building with.